pipeline/runtime/runtestsV2.go

```diff
@@ -106,6 +106,9 @@ func executeRunTestsV2Step(ctx context.Context, f RunFunc, r *api.StartStepReque
 	outputFile := fmt.Sprintf("%s/%s-output.env", pipeline.SharedVolPath, step.ID)
 	step.Envs["DRONE_OUTPUT"] = outputFile
 
+	outputSecretsFile := fmt.Sprintf("%s/%s-output-secrets.env", pipeline.SharedVolPath, step.ID)
```

Comment:
Same for this

Reply:
observed the following support in the file . so added logic for secrets here also . can be removed if not required
`step.Envs["DRONE_OUTPUT"] = outputFile`

Reply:
I think let's do for run.go for now only ? If the requirement comes for runtest we can add that later ?